scipy/sparse/linalg/_eigen/arpack/arpack.py

```diff
@@ -36,6 +36,7 @@
 # for eigenvalues by providing a shift (sigma) and a solver.
 
 import numpy as np
+import threading
```

Comment:
was this just missing before and the issue wasn't detected? just asking because I don't see it used in the diff here